frontend/src/metabase-lib/Question.ts

```diff
@@ -494,8 +486,9 @@ class Question {
 
     const hasSinglePk =
       table?.fields?.filter(field => field.isPK())?.length === 1;
+    const isStructured = !Lib.queryDisplayInfo(this.query()).isNative;
```

Comment:
In some files we're introducing `isStructured` and in some files we're just plainly using `!isNative`. I think we should go with 1 approach:
- we can get rid of `isStructured` and always use `!isNative`
- we can fully embrace `isStructured` by adding it to `Lib.queryDisplayInfo` result

WDYT?

----

_this is a non-blocking comment_

Reply:
I'm for the first: `!isNative`

Reply:
https://github.com/metabase/metabase/pull/37977/commits/b9e8074472138a7821eafe77d7d75d948a580017

Reply:
Addressed in this commit
fdc93b1809862f4a4a97d508f9282e7870e6b78f

Reply:
After some weird CI failures that I couldn't reproduce locally, I moved this step to a separate PR.
https://github.com/metabase/metabase/pull/38199